client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/sidebar.tsx

```diff
@@ -13,10 +13,22 @@ import { defaultSiteDetails, buildSiteDetails } from './lib/fixtures';
 const siteName = 'testlinkinbio';
 const secondAndTopLevelDomain = 'wordpress.com';
 const siteSlug = `${ siteName }.${ secondAndTopLevelDomain }`;
-const sidebarURL = `https://${ siteName }.${ secondAndTopLevelDomain }`;
+const sidebarFreeDomainObject = {
```

Comment:
We might want to create a new test fixture for this data https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/lib/fixtures.ts